istio/tests/terraform/script.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/python
+
+# Write it as a python script for portability
+import glob
+import os
+from subprocess import check_call
+
+version = os.environ['ISTIO_VERSION']
+opj = os.path.join
+
+# We don't care about the platform as we only use yaml files
+check_call(
+    [
+        "curl",
+        "-o",
+        "istio.tar.gz",
+        "-L",
+        "https://github.com/istio/istio/releases/download/{version}/istio-{version}-linux.tar.gz".format(
+            version=version
+        ),
+    ]
+)
+check_call(["tar", "xf", "istio.tar.gz"])
+
+check_call(["kubectl", "create", "ns", "istio-system"])
+check_call(["kubectl", "label", "namespace", "default", "istio-injection=enabled"])
+
+istio = "istio-{}".format(version)
+
+for f in glob.glob(opj([istio, "install", "kubernetes", "helm", "istio-init", "files", "crd*.yaml"])):
```

Comment:
The python script fails for me, `os.path.join` should be used with unpacked arguments

```suggestion
for f in glob.glob(opj(istio, "install", "kubernetes", "helm", "istio-init", "files", "crd*.yaml")):
```

Reply:
Sorry about that...